_posts/2016-09-22-professional-development-at-artsy-engineering.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Professional Development at Artsy Engineering"
+date: 2016-09-22 16:37
+comments: true
+author: alan
+categories: [engineering, professional, development, organization, education]
+---
+
+In considering an offer to join us at Artsy, one of our newest incoming engineers asked me a great question: _How does the tech team do professional development?_
+
+As I thought about it, I began to realize that the answer is “a lot”! Most of our efforts evolved organically. Someone had an idea, and people rallied around it. I thought it would be useful to share, in case others are inspired by what's caught on here. Here are some of the things we do.
+
+<!-- more -->
+
+# Practices
+
+Our engineering team used to be organized along _practice_ lines: web, mobile native, and platform. Following our recent [product reorganization][1], we’re primarily organized in product teams, which are more deeply integrated and aligned with our business units. But practices are here to stay. They serve as horizontal channels for developers working in similar technologies to collaborate and share best practices.
+
+# Collaboration
+
+In day-to-day work, we do a lot of pair programming and whiteboard architecture on an ad hoc basis. This happens between teams as much as within them.
+
+# Lunch ’n’ Learn
+
+Every Thursday, we do a Lunch-and-Learn session. Historically, we mostly showed off tech we use internally. But over about a year, most of our tech stack has been presented this way, so we also bring in engineers we know at other companies to share what they’re working on.
+
+# Conferences
+
+Devs are encouraged to attend conferences. The engineering budget covers expenses for 1 per year as an attendee, and unlimited as a presenter. Many of our engineers are experienced presenters -- and I know [Orta][1] has even been a conference organizer. So there’s plenty of support for first-time presenters. Lunch ’n’ Learn sessions have served as a place to do a dry run for a presentation.
+
+# Community engagement
+
+Professional development isn’t just about absorbing new information, it’s also about teaching and sharing. That’s one of the best ways to truly master a topic. We encourage folks to participate in [OSS][2], blogging (obviously), speaking, and other types of community engagement, and we’ve got really experienced people on all those fronts to support for helping make this happen.
+
+# Slack discussions
+
+We’ve got an active #tech-learning Slack channel, which people use to share and discuss articles. We also have a number of language-specific Slack channels, such as #elixir, #swift, and #scala (my personal fave).
+
+# Mentorship
+
+Newer devs are paired with a mentor, to be a point person in helping along their professional development.
```

Comment:
@ashfurrow  wanna take a stab?

My general gists are:

- mentor relationship help improve both sides
- specific relationships means there's always someone you can talk to as a newbie
- specific relationship means that mentor understands it's a part of work time
- there is also the artsy company mentor thing, which I've forgotten the name of - where someone outside your field helps you onboard